src/waypoint_cmd.cpp

```diff
@@ -292,6 +319,147 @@ CommandCost CmdBuildRailWaypoint(DoCommandFlag flags, TileIndex start_tile, Axis
 	return cost;
 }
 
+/**
+ * Convert existing road to waypoint. Eg build a waypoint station over
+ * piece of road
```

Comment:
```suggestion
 * Build a road waypoint on an existing road.
```
The other lines in this function documentation are missing periods too.